x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -75,15 +75,29 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await loggingMessageInput.clearValue();
       await loggingMessageInput.type('test message');
 
-      await testSubjects.click('slackAddVariableButton');
-      const variableMenuButton = await testSubjects.find('variableMenuButton-0');
-      await variableMenuButton.click();
+      // TODO: uncomment variables test when server API will be ready
+      // await testSubjects.click('slackAddVariableButton');
+      // const variableMenuButton = await testSubjects.find('variableMenuButton-0');
+      // await variableMenuButton.click();
 
-      await testSubjects.click('selectIndexExpression');
+      await testSubjects.click('selectIf.index-threshold-SelectOptionndexExpression');
```

Comment:
Is this a typo? Or is it supposed to be this? Not sure as I can't find anything named either, but feels like this should be: `SelectOptionIndexExpression`